pkg/server/api/bundle/v1/service.go

```diff
@@ -518,3 +569,50 @@ func applyBundleMask(b *types.Bundle, mask *types.BundleMask) {
 		b.JwtAuthorities = nil
 	}
 }
+
+func fieldsFromBundleProto(proto *types.Bundle, inputMask *types.BundleMask) logrus.Fields {
+	fields := logrus.Fields{
+		"trust_domain": proto.TrustDomain,
+	}
+
+	if inputMask == nil || inputMask.RefreshHint {
+		fields["refresh_hint"] = proto.RefreshHint
+	}
+
+	if inputMask == nil || inputMask.SequenceNumber {
+		fields["sequence_number"] = proto.SequenceNumber
+	}
+
+	if inputMask == nil || inputMask.JwtAuthorities {
+		for k, v := range fieldsFromJwtAuthoritiesProto(proto.JwtAuthorities) {
+			fields[k] = v
+		}
+	}
+
+	if inputMask == nil || inputMask.X509Authorities {
+		for k, v := range fieldsFromX509AuthoritiesProto(proto.X509Authorities) {
+			fields[k] = v
+		}
+	}
+	return fields
+}
+
+func fieldsFromJwtAuthoritiesProto(jwtAuthorities []*types.JWTKey) logrus.Fields {
+	fields := logrus.Fields{}
+	for i, jwtAuthoriry := range jwtAuthorities {
+		fields[fmt.Sprintf("jwt_authority_expires_at_%d", i)] = jwtAuthoriry.ExpiresAt
```

Comment:
nit: could we use `.` rather than `_` as the separator between the property name and the index? e.g. `jwt_authority_expires_at.0`, `jwt_authority_expires_at.1`, etc. I think it more clearly distinguishes the fact that this field is part of a flattened list of values